recipes/bregman-learning/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "bregman-learning" %}
+{% set version = "0.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bregman_learning-{{ version }}.tar.gz
+  sha256: 47f2e2bae4a985c1cd4e12c4c058cbac90c70746c74647843ef247b1998ccbb1
+
+build:
+  skip: true  # [py<310]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - setuptools >=30.3.0
+    - wheel
+    - pip
+  run:
+    - python
+    - pytorch >=1.12.0
+    - numpy ==1.26  # [win]
+    - numpy >=1.22  # [not win]
+    - igraph
```

Comment:
I believe this should be `python-igraph` instead of `igraph`

Reply:
That was it.😄